<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWidget</class>
 <widget class="QWidget" name="RegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>560</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nickname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>731</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>140</y>
     <width>861</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : red</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>620</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password2Edit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Repeat password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fnameEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnameEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
